Extensions/PlatformBehavior/tests/platformerobjectruntimebehavior.spec.js

```diff
@@ -2127,7 +2109,7 @@ describe('gdjs.PlatformerObjectRuntimeBehavior', function () {
       platform2.setAngle(-30);
       platform2.setPosition(
         platform.getX() + platform.getWidth() + vertexDeltaX,
-        platform.getY() + vertexDeltaY
+        platform.getY() + vertexDeltaY - 0.5
       );
```

Comment:
I remember checking with IDE that the rotated platform was corner to corner with the other platform. If the test only works when the platform is lowered a bit, it's revealing of an issue. The platformer character will probably be stuck.

Reply:
True, changing this might be concerning. Or maybe this was just to ensure the position is ok after that, in which case @ClementPasteau it would be better to not move the platform, and instead adapt the position of the object.

In other words, the changes in the tests should not have any change in the position of the platforms, because levels of games won't change. There might a slight change in the object position (due to the rounding being gone), but that should be it (in theory at least).